x-pack/examples/lens_embeddable_inline_editing_example/public/utils.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { DataView } from '@kbn/data-views-plugin/public';
+import type {
+  LensConfig,
+  LensConfigOptions,
+} from '@kbn/lens-embeddable-utils/config_builder/types';
+
+export const getConfigOptions = (dataView: DataView, isESQL?: boolean) => {
+  const index = dataView.getIndexPattern();
+  const timeFieldName = dataView.getTimeField()?.name;
+  if (isESQL) {
+    return {
+      config: {
+        chartType: 'metric',
+        title: 'metric chart',
+        dataset: {
+          esql: `from ${index} | stats count=count()`,
+        },
+        value: 'count',
+      } as unknown as LensConfig,
+      options: {
+        embeddable: true,
+        timeRange: {
+          from: 'now-30d',
+          to: 'now',
+          type: 'relative',
+        },
+        query: {
+          esql: `from ${index} | stats count=count()`,
+        },
+      } as unknown as LensConfigOptions,
```

Comment:
Yes and is not so simple because it starts from the Lens side. So we first need to fix the types there and then update the builder. Not a task for this PR but I will create a tech debt issue to track this.

Reply:
Added this to the code quality tasks https://github.com/elastic/kibana/discussions/161434